react/components/ShippingOptionList.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import { Address, DeliveryOption } from 'vtex.checkout-graphql'
+import { ButtonPlain, IconEdit } from 'vtex.styleguide'
+import { PlaceDetails } from 'vtex.place-components'
+import { TranslateEstimate } from 'vtex.shipping-estimate-translator'
+import { FormattedPrice } from 'vtex.formatted-price'
+import { OrderShipping } from 'vtex.order-shipping'
+import { FormattedMessage } from 'react-intl'
+import { ListGroup, GroupOption } from 'vtex.checkout-components'
+
+interface Props {
+  deliveryOptions: DeliveryOption[]
+  onEditAddress?: () => void
+  onEditReceiverInfo?: () => void
+  onDeliveryOptionSelected?: (id: string) => void
+  selectedAddress: Address
+}
+
+const ShippingOptionList: React.FC<Props> = ({
+  deliveryOptions,
+  onEditAddress,
+  onEditReceiverInfo,
+  onDeliveryOptionSelected = () => {},
+  selectedAddress,
+}) => {
+  const handleDeliveryOptionSelect = (deliveryOption: DeliveryOption) => {
+    onDeliveryOptionSelected(deliveryOption.id!)
+  }
+
+  return (
+    <div>
+      {selectedAddress.receiverName && (
+        <div className="c-muted-1">
+          <span className="fw6 flex items-center">
+            <FormattedMessage id="store/checkout.shipping.receiverLabel" />{' '}
+            <div className="dib ml4">
+              <ButtonPlain onClick={onEditReceiverInfo}>
+                <IconEdit solid />
+              </ButtonPlain>
+            </div>
+          </span>
+
+          <div className="mt2 mb6 lh-copy">{selectedAddress.receiverName}</div>
+        </div>
+      )}
+
+      <span className="t-body fw6 flex items-center">
```

Comment:
`span` elements are of type `inline` and should therefore not have block elements as children. Won't break the markup but it's semantically wrong.

Reply:
do you suggest we change this to `p` tag instead?

Reply:
No, it would be similar. The content semantically allowed for the `p` tag is the phrasing one: https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#Phrasing_content 
This `span` should  be a `div`.

Reply:
ohh, nice. so, I _could_ change this to a `p` tag because the only other thing inside here is the `Button` (which is only wrapped in a `div` because styleguide doesn't permit `className` props)